transmartApp/grails-app/init/org/transmartproject/app/BootStrap.groovy

```diff
@@ -50,11 +50,6 @@ class BootStrap {
 
         // force marshaller registrar initialization
         grailsApplication.mainContext.getBean 'marshallerRegistrarService'
-
-        if ('clientCredentialsAuthenticationProvider' in
-                grailsApplication.config.grails.plugin.springsecurity.providerNames) {
-            OAuth2SyncService.syncOAuth2Clients()
-        }
     }
 
```

Comment:
The blocks about SAML and configFine probably also should be in transmart-server. I don't know what the purpose of the forceEagerSessionCreation and concurrent sessions stuff is, so I'm not sure if it belongs here or in -server.